core/arch/arm/kernel/kern.ld.S

```diff
@@ -53,9 +53,27 @@
 OUTPUT_FORMAT(CFG_KERN_LINKER_FORMAT)
 OUTPUT_ARCH(CFG_KERN_LINKER_ARCH)
 
+/*
+ * Start coherent RAM with a platform specific structure.
+ * Generic data to be place in coherent RAM can use attribute "coherent_ram".
+ */
+#define DECLARE_COHERENT_RAM_SECTION						\
```

Comment:
To add some flexibility regarding the location of the coherent RAM, in case one wants it at a dedicated place. I proposed to possible location: before the load address or after the core reserved vaddr range (virt, no phys only, to fit with pager constraints). Since there were those 2 places, I preferred to use a single macro. You feel it bring complexity ?